OOP/Assignment1/MatrixTest.java

```diff
@@ -0,0 +1,124 @@
+import static org.junit.Assert.*;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class MatrixTest {
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+	}
+
+	@AfterClass
+	public static void tearDownAfterClass() throws Exception {
+	}
+
+	@Before
+	public void setUp() throws Exception {
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	Matrix matrix = new Matrix(4, 4);
+
+	@Test
+	public void testAddElements1() {
+
+		matrix.addElements(1, 1, 2);
+		assertEquals(2, matrix.data[1][1]);
+	}
+
+	@Test
+	public void testAddElements2() {
```

Comment:
ACTION: You should not test for those fields/methods which are not public (until or unless specified).